File: PPJ/PPJ ćwiczenia/PPJ/kod/WIs 21c/Ppj20/1,2,3/Osoba.java
public class Osoba {
    String imie, nazwisko;
    int rokUrodzenia;

    public Osoba(String imie, String nazwisko, int rokUrodzenia) {
        this.imie = imie;
        this.nazwisko = nazwisko;
        this.rokUrodzenia = rokUrodzenia;
    }

    public String toString() {
        return imie + " " + nazwisko + " (" + rokUrodzenia + ")";
    }
}

File: MAS/MAS_s20960_MP3/Dynamiczne/Osoba.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MAS_s20960_MP3.Dynamiczne
{
    internal class Osoba
    {
        public string Imie { get; set; }
        public string Nazwisko { get; set; }

        public Osoba(string imie, string nazwisko)
        {
            Imie = imie;
            Nazwisko = nazwisko;
        }

        public override string ToString()
        {
            return this.GetType().Name + ": " + Imie + " " + Nazwisko;
        }
    }
}
